internal/config/db.go

```diff
@@ -59,6 +59,9 @@ type Postgres struct {
 type PostgresTLS struct {
 	CABundlePath string `yaml:"ca_bundle_path" json:"ca_bundle_path" env:"CA_BUNDLE_PATH"`
 	Enable       bool   `yaml:"enable" json:"enable" env:"ENABLE"`
+	// SslMode should be one of the postgresql ssl modes. Default to 'verify-full'.
+	// Ref: https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-SSLMODE-STATEMENTS
+	SslMode string `yaml:"ssl_mode" json:"ssl_mode" env:"SSL_MODE"`
```

Comment:
Why not give the user more control over all options using an "Options" string?

Reply:
Interesting idea, you are thinking about **all** options, or just all TLS/SSL options? https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PARAMKEYWORDS

if we consider all options (so in `Postgres`, not `PostgresTLS`), those options would be appended to the DSN and passed to postgresql. Should we validate that the user doesn't specify any option that koko already generate, or we are good to just delegate this validation to postgres.

Reply:
A similar thing actually introduced with the MySQL changes, just didn't give the capability to Postgres. Though some care needed to be done, as certain options could break things:
https://github.com/Kong/koko/blob/8fdb6c3974453983299c92198e6c55c6bf5b06e7/internal/persistence/mysql/options.go#L20

Reply:
Give users as many options as possible without making it easy for them to shoot themselves.

Reply:
Let's go that direction then. I will come up with something similar to MySQL.